From bd1988c40bc293c5beb585ed67a4ca9d9127a2d4 Mon Sep 17 00:00:00 2001
From: Nicolas Frattaroli <nicolas.frattaroli@collabora.com>
Date: Tue, 20 May 2025 20:50:09 +0200
Subject: [PATCH 13/42] arm64: dts: rockchip: rename rk3588-rock-5b.dtsi

As subsequent patches will add ROCK 5T support, rename the .dtsi file to
reflect that it's shared between ROCK 5B, ROCK 5B+ and ROCK 5T.

This is done separately from moving the 5B and 5B+ only nodes to a
common tree so that the history stays bisectable and the diff easily
reviewable.

Signed-off-by: Nicolas Frattaroli <nicolas.frattaroli@collabora.com>
---
 .../{rk3588-rock-5b.dtsi => rk3588-rock-5b-5bp-5t.dtsi}         | 0
 arch/arm64/boot/dts/rockchip/rk3588-rock-5b-plus.dts            | 2 +-
 arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts                 | 2 +-
 3 files changed, 2 insertions(+), 2 deletions(-)
 rename arch/arm64/boot/dts/rockchip/{rk3588-rock-5b.dtsi => rk3588-rock-5b-5bp-5t.dtsi} (100%)

diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dtsi b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-5bp-5t.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3588-rock-5b-5bp-5t.dtsi
diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-plus.dts b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-plus.dts
index 4096cd853..be4b0acdb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b-plus.dts
@@ -2,7 +2,7 @@
 
 /dts-v1/;
 
-#include "rk3588-rock-5b.dtsi"
+#include "rk3588-rock-5b-5bp-5t.dtsi"
 
 / {
 	model = "Radxa ROCK 5B+";
diff --git a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
index afe5781ea..fea61074b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
@@ -2,7 +2,7 @@
 
 /dts-v1/;
 
-#include "rk3588-rock-5b.dtsi"
+#include "rk3588-rock-5b-5bp-5t.dtsi"
 
 / {
 	model = "Radxa ROCK 5B";
-- 
2.39.5

